tests/python/relay/test_pass_plan_devices.py

```diff
@@ -0,0 +1,1405 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Now this is a test file!

Reply:
Not to rain on your parade but it might be worth directly using text format here?

Reply:
Yeah it's pretty awful and I nearly switched a few times already. Which format should I use? Can it capture the attributes?

Reply:
+1 for relay text format, i think that would greatly help

Reply:
Done.